tests/adapter_tests/falcon/test_falcon.py

```diff
@@ -18,6 +20,13 @@
 from tests.utils import remove_os_env_temporarily, restore_os_env
 
 
+def new_falcon_app():
+    if falcon.version.__version__.startswith("2."):
+        return falcon.API()
```

Comment:
`falcon.API` is deprecated in 3.x. However, the new recommended way `falcon.App` does not exist in 2.x